client/post-editor/editor-diff-viewer/style.scss

```diff
@@ -0,0 +1,20 @@
+.editor-diff-viewer__title {
+	padding-left: 10px; /* Intentionally non-standard to align editor body margin (10px) */
+	padding-right: 10px;
+	font-family: $serif;
+	font-size: 28px;
+	color: $gray-dark;
+	font-weight: 600;
+	resize: none;
+	-ms-overflow-y: hidden !important;
```

Comment:
copy-pasted from the the editor's title input :) some (all? :D ) are definitely useless.